sotodlib/site_pipeline/update_det_cal.py

```diff
@@ -617,6 +620,28 @@ def find_correction_results(band, chan, dset):
             else:
                 cal.phase_to_pW = pA_per_phi0 / (2 * np.pi) / cal.s_i * cal.polarity
 
+            # Add bandpass informaton from bias group
+            bgs_lowband = [0, 1, 4, 5, 8, 9]
+            bgs_highband = [2, 3, 6, 7, 10, 11]
+
+            tube_flavor = am.obs_info.tube_flavor
+            if tube_flavor == 'mf':
+                lowband_str = 'f090'
+                highband_str = 'f150'
+
+            elif tube_flavor == 'uhf':
+                lowband_str = 'f220'
+                highband_str = 'f280'
+
+            elif tube_flavor == 'lf':
+                lowband_str = 'f030'
+                highband_str = 'f040'
+
+            if cal.bg in bgs_lowband:
+                cal.bandpass = lowband_str
+            elif cal.bg in bgs_highband:
+                cal.bandpass = highband_str
```

Comment:
How about something more compact like this? 
```
bgs = {'lb':[0, 1, 4, 5, 8, 9], 'hb':[2, 3, 6, 7, 10, 11]}
tube_flavor = am.obs_info.tube_flavor
band_str = {'mf':{'lb':'f090', 'hb':'f150'},
            'uhf':{'lb':'f220', 'hb':'f280'},
            'lf':{'lb':'f030', 'hb':'f040'}}
if cal.bg in bgs['lb']:
    cal.bandpass = band_str[tube_flavor]['lb']
elif cal.bg in bgs['hb']:
    cal.bandpass = band_str[tube_flavor]['hb']
```

And maybe `bgs` and `band_str` can be defined once/globally at the top of the script.